vyper/semantics/types/function.py

```diff
@@ -344,7 +344,11 @@ def from_vyi(cls, funcdef: vy_ast.FunctionDef) -> "ContractFunctionT":
 
         return_type = _parse_return_type(funcdef)
 
-        if len(funcdef.body) != 1 or not isinstance(funcdef.body[0].get("value"), vy_ast.Ellipsis):
+        body = funcdef.body
+
+        if len(body) != 1 or not (
+            isinstance(body[0], vy_ast.Expr) and isinstance(body[0].get("value"), vy_ast.Ellipsis)
```

Comment:
once we have refined the type of `body[0]`, we don't need to use `get()`, we can just directly reference `.value`.

Reply:
addressed in: https://github.com/vyperlang/vyper/pull/4177/commits/f006571118b7d48462c0815b49f092a0c30968c3